source/PlayerInfo.cpp

```diff
@@ -1346,6 +1346,7 @@ void PlayerInfo::ReorderShip(int fromIndex, int toIndex)
 	shared_ptr<Ship> ship = ships[fromIndex];
 	ships.erase(ships.begin() + fromIndex);
 	ships.insert(ships.begin() + toIndex, ship);
+	ships[0]->SetIsParked(false);
```

Comment:
Is it intended that the ship gets unparked even if it can't be set as flagship (for example is in another system or is an automaton)?

Reply:
For my understanding it is, isnt it? I tested it between the systems Rutilicus & Menkent. Before this change i was able to unpark them manually even they had been in another system. So i thought this was intended. So far my knowledge about flagships that cant be unparked due to other conditions is lacking at this point. For example automaton is unfortunatly new to me.

Reply:
No I mean you can unpark a ship that is in another system, but it won't become your flagship immediately. I thought the original request was to unpark the first ship, because the player has placed it there to make it the flagship.

Reply:
Ah I understand what you mean. It works beeing in the same system as intended. But not in a different system, you are right. I have checked it again. It seems that once I unpark any ship from a different system, on the first launch no unparked ship appears. I need to land and depart again with no change at all for the unparked ships to appear. 
---Is this supossed to behave like that? 

Anyway the flagship is not swapped this way in a different system. But it is unparked. I suppose this has to do with somewhere else? 

But is the initiate change worth a merge, as it somehow manages to fix a little step of it? I could take a closer for the other behavior. But I am not sure if I can do it in time. As this is my study project of just a week my professor would grant me 10% for an accepted merge request :)

Reply:
> It seems that once I unpark any ship from a different system, on the first launch no unparked ship appears. I need to land and depart again with no change at all for the unparked ships to appear.
> ---Is this supossed to behave like that?

If you unpark a ship that is in a different system than yours, it may need some time to travel to you.